### ISSUE TITLE
Plugin does not support Jest 28

### ISSUE THREAD

**Issue by fdc-viktor-luft** [alangpierce/sucrase#694]
Error when running `jest@28.0.1` with configuration:

```js
transform: { "\\.(js|jsx|ts|tsx)$": "@sucrase/jest-plugin" },
```

```
ERROR: ● Invalid return value:
  Code transformer's `process` method must return an object containing `code` key 
  with processed string. If `processAsync` method is implemented it must return 
  a Promise resolving to an object containing `code` key with processed string.
  Code Transformation Documentation:
  https://jestjs.io/docs/code-transformation
```

**Comment by xalisys** [comment]
I am facing the exact same issue, any ideas?

**Comment by fdc-viktor-luft** [comment]
Yes. Luckily it is very easy to fix. I'll create a PR.

**Comment by alangpierce** [comment]
@fdc-viktor-luft Thanks for the PR! I just published the fix as @sucrase/jest-plugin 2.2.1
